src/site/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #
 # VERSION	1.0
 
-FROM adoptopenjdk:11-jdk-hotspot
+FROM eclipse-temurin:11-jre-focal
```

Comment:
We IMO need a JDK to run maven?

Reply:
You're right,

Reply:
![Screenshot from 2021-09-10 12-53-04](https://user-images.githubusercontent.com/81145350/132806655-f8b07af7-027d-4609-9307-a8d4299bdbb2.png)

Reply:
```suggestion
FROM adoptopenjdk:11-jdk-hotspot
```

Reply:
Can't we use `eclipse-temurin:11-jdk-focal` ?

Reply:
updated `eclipse-temurin:11-jdk-focal`